web/forms.py

```
@@ -1677,3 +1681,42 @@ class Meta:
+            "slug": TailwindInput(),
+            "description": TailwindTextarea(attrs={"rows": 3}),
+            "features": TailwindJSONWidget(),
+            "price_monthly": TailwindNumberInput(attrs={"step": "0.01", "min": "0"}),
+            "price_yearly": TailwindNumberInput(attrs={"step": "0.01", "min": "0"}),
+            "billing_period": TailwindSelect(),
+            "stripe_monthly_price_id": TailwindInput(),
+            "stripe_yearly_price_id": TailwindInput(),
+            "is_active": TailwindCheckboxInput(),
+            "is_popular": TailwindCheckboxInput(),
+            "order": TailwindNumberInput(attrs={"min": "0"}),
+        }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)